VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKTrackingEvent.h

```diff
@@ -0,0 +1,24 @@
+@import JSONModel;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface VSDKTrackingEvent : JSONModel
+
+/**
+  Client identifier (REQUIRED)
+ */
+@property (nonnull) NSString *clientId;
```

Comment:
> Is it required nonnull? Shouldn't NS_ASSUME_NONNULL_BEGIN be enough to prevent null assignments to variables declared in this region?

No, it is not required, but I prefer to remove the `NS_ASSUME_NONNULL_BEGIN` in all model classes (to make it more explicit to the developer which fields are optional and which fields aren't)

> Also, can't we also mark the variable as nonatomic?

 What benefit do you see in that?

Reply:
> No, it is not required, but I prefer to remove the `NS_ASSUME_NONNULL_BEGIN` in all model classes (to make it more explicit to the developer which fields are optional and which fields aren't)

👍 





> > Also, can't we also mark the variable as nonatomic?
> 
> What benefit do you see in that?

[Avoid thread safety boilerplate](https://stackoverflow.com/a/589348)

Reply:
During an offline conversation, we decided not to do this